test/unit/alignment/pairwise/fixture/alignment_fixture.hpp

```diff
@@ -16,14 +16,38 @@ namespace seqan3::test::alignment::fixture
 
 static constexpr auto INF = std::nullopt;
 
-static constexpr auto NON = detail::trace_directions::none;
-static constexpr auto D = detail::trace_directions::diagonal;
-static constexpr auto U = detail::trace_directions::up;
-static constexpr auto L = detail::trace_directions::left;
-static constexpr auto DU = D | U;
-static constexpr auto UL = U | L;
-static constexpr auto DL = D | L;
-static constexpr auto DUL = D | U | L;
+static constexpr auto N     = detail::trace_directions::none;
+static constexpr auto D     = detail::trace_directions::diagonal;
+static constexpr auto u     = detail::trace_directions::up;
+static constexpr auto l     = detail::trace_directions::left;
```

Comment:
why those changes?

Reply:
Because we changed the ascii representation for gap extension from capital letter to small letter. The capital letter is used as a visual hint for gap open. But before this was only tailored to the edit distance algorithm and after I introduced the concept of gap extension and gap opening we adapted it accordingly. Thus I made it consistent within the tests as well, so that the name represents the actual ascii representation of the trace directions.